docker-compose-rule-core/src/integrationTest/java/com/palantir/docker/compose/AggressiveShutdownStrategyIntegrationTest.java

```diff
@@ -29,20 +29,20 @@
 
     @Test
     public void shut_down_multiple_containers_immediately() throws Exception {
-        DockerComposeRule rule = DockerComposeRule.builder()
+        DockerComposeManager docker = DockerComposeManager.testBuilder()
```

Comment:
shall we move all these tests into docker-compose-rule-core, since they're not using the rule anymore but only the manager?

Reply:
They are in docker-compose-rule core!